### PR TITLE
[SEO] Add Tafsir by Id page tags

### PR DESCRIPTION
**Pull request by osamasayed**
### Summary
This PR adds Tafsirs route e.g. `/77:1/tafsirs/169` to the sitemap and also the meta tags for that route.

### CODE REVIEW

**Comment by mmahalwy** [comment]
#### 📝 Changed routes:


| [**…Id\]/tafsirs/\[tafsirId\]**](https://quran-com-1r1lr4342-qurancom.vercel.app/[chapterId]/tafsirs/[tafsirId]) |
|:-:|
|<a href="https://quran-com-1r1lr4342-qurancom.vercel.app/[chapterId]/tafsirs/[tafsirId]">Screenshot of /[chapterId]/tafsirs/[tafsirId]</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/m5ZH7vKMk8FxldGtinp-stAEIwbYnwS9NSShSlqIPKGWGgKDG920va8i8buQ8Cy82VjG90hrJdKDUMugwXE-nw.png">(view full size)</a> |







Commit 9094e6fde66541fb288fac565afb4f929dfdf2d3 (https://quran-com-1r1lr4342-qurancom.vercel.app).

File: next-sitemap.js [hunk 1]
```diff
@@ -72,6 +86,10 @@ module.exports = {
   },
   additionalPaths: async (config) => {
     const result = [];
+    let tafsirIDs = [];
+    await getTafsirsIds().then((response) => {
+      tafsirIDs = response.tafsirs.map((tafsir) => tafsir.id);
```
Comment (naveed-ahmad) [inline next-sitemap.js:91]: we can use slugs instead of tafsir id

https://staging.quran.com/api/qdc/resources/tafsirs

`en-tafisr-ibn-kathir` or 169?

Comment (naveed-ahmad) [reply next-sitemap.js:91]: "Get tafsir api" support both ids and slugs 

https://staging.quran.com/api/qdc/tafsirs/en-tafisr-ibn-kathir/by_ayah/1:2?words=true&fields=text_uthmani

Comment (osamasayed) [reply next-sitemap.js:91]: @naveed-ahmad JazakAllah Kahiran, definitely the slugs are better but need to handle them first in the route. Thanks for letting me know that we can use the slugs!

**Pull request closed without merging**